pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/MultiStageBrokerRequestHandler.java

```diff
@@ -128,7 +128,14 @@ protected BrokerResponse handleRequest(long requestId, String query, SqlNodeAndO
       Long timeoutMsFromQueryOption = QueryOptionsUtils.getTimeoutMs(queryOptions);
       queryTimeoutMs = timeoutMsFromQueryOption != null ? timeoutMsFromQueryOption : _brokerTimeoutMs;
       database = DatabaseUtils.extractDatabaseFromQueryRequest(queryOptions, httpHeaders);
-      QueryEnvironment queryEnvironment = new QueryEnvironment(database, _tableCache, _workerManager);
+      boolean useImplicitColocatedByDefault = _config.getProperty(CommonConstants.Broker.IMPLICIT_COLOCATE_JOIN,
+          CommonConstants.Broker.DEFAULT_IMPLICIT_COLOCATE_JOIN);
+      QueryEnvironment queryEnvironment = new QueryEnvironment(QueryEnvironment.configBuilder()
+          .database(database)
```

Comment:
(nit, format) This doesn't follow the `Pinot Style`. Either reformat it or use comment to turn formatter off. Same for other places if custom format is more readable. For this specific one since there are not that many values, I personally prefer keeping them one line

Reply:
I don't think this is against Pinot Style (ie checkstyle doesn't fail), it may be against the default config in intellij. I can add this format off, but I think we should improve that default config to support both styles.

I have to say that it is very difficult to me to read this large lines with several chained method called.

Reply:
Applied formatter off